### ISSUE TITLE
Expected A4 (multiple choice) accuracy on validation set

### ISSUE THREAD

**Issue by ruochiz** [A2Zadeh/Social-IQ#11]
Hi, 

Could you help to give a rough estimation on the validation accuracy (A4) that the tmfn_bert can get on multiple choice QA? I was trying to adapt the code provided in this repo from binary to multiple choice QA, just want to make sure the adaption doesn't go wrong. Thanks!
